main.py

```diff
@@ -0,0 +1,17 @@
+from flask import Flask
+app = Flask(__name__)
+
+@app.route('/')
```

Comment:
This is good for testing, but you want to avoid checking-in (merging) code where the app does not have a specific URL, even if it is for testing. This can cause issues when you are trying to go back and find out why your port may be hogged by a different script.